docs/configuration/janitor-provider.md

```
@@ -0,0 +1,221 @@
+The Janitor Provider serves gRPC over TLS. The certificate is issued and renewed by cert-manager, which is a required dependency.
+
+```yaml
+janitor-provider:
+  tls:
+    enabled: true
+    certDir: "/etc/nvsentinel/janitor-provider/tls"
+    issuerName: "janitor-selfsigned-issuer"
+    secretName: ""
+```
+
+#### enabled
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)